src/content/api.js

```diff
@@ -26,9 +26,14 @@ const testUser = {
 
 
 const chrome = window.chrome;
+const appConstants = require('../app-constants.json');
 
 function sendMessage(data, callback) {
-  chrome.runtime.sendMessage("lcpmicjjmgdclbmaeeeffjkiffpgddig", data, callback)
+  if (appConstants.isProdMod) {
```

Comment:
IsProdMode is not necessary because you can just check appConstants.googleExtensionId